articles/detecting-botnets-using-dns-based-technique/index.md

```diff
@@ -0,0 +1,65 @@
+Over the past few years, more people and organizations have embraced the use of technology in day-to-day lives, by use of simple networking to complex networking technologies that enable data sharing and communication. Data sharing has come with its security issues challenges. Data security challenges have been necessitated by botnets, used by cybercriminals to instigate various attacks. 
+
+This article pursues the DNS-based approaches for detecting botnets and also explains why these approaches are widely used by providing examples of DNS-based tools.
+
+### What is a Botnet?
+A botnet can be defined as a group of computers or digital devices that are infected with malware to enable hackers to control them. Botnets are used by cybercriminals to instigate attacks such as unauthorized access of information, data theft, credentials leak, and DDoS attacks. To be able to mitigate such a security threat, one should be in a position to identify such attacks in a network. 
+Botnets conceal their malicious activities and avoid detection by using a number of internet protocols. In the recent past, hackers have abused multiple protocols, whilst DNS has become the central target of cyberattacks.  Such an attack is the advanced persistent threat (APT). In most cases, experienced botnet developers avoid using domain names and instead, they have successfully set up P2P botnets. 
+
+In this regard, the DNS system is used by such botnets to support their command and control infrastructure. Further, the use of ‘domain generation algorithms (DGAs)’ in botnet malware has risen in the past few years. This has compounded the difficulty in the detection of botnets partially due to the easiness of developing and administrating botnets based on DNS.  
+
+The following diagram shows the structure of a botnet. 
+![alt text](botnet-structure.png)
```

Comment:
Please use the right image format as per Section's guidelines. Follow this structure: ```![image title](/engineering-education/new-folder-name/image-name.jpg)```